src/browser/api/notifications/subscriptions.ts

```diff
@@ -391,15 +391,16 @@ function genAnimationFunction(defaultTop: number, numNotes: number): (noteWin: a
     return (noteWin: any, idx: number) => {
         const {name, uuid} = noteWin;
         const identity: Identity = {name, uuid};
+        const opacity = noteWin.opacity || 1;
```

Comment:
:neckbeard: 
• can simply put it below into the object (optional)
• can notifications be transparent (opacity: 0)? If yes, than this won't work, just asking...

Reply:
- don't really have opinion, happy to make either way my default but seems like I see it defined as variable much more often in our code
- Tried to change as little as possible, before this change all notifications were set to opacity 0 and this function would take them from 0 to 1 over time.  This way keeps that behavior constant.  Do we want people to be able to set opacity 0?

Reply:
I don't know how other stuff in notifications work, just if the `opacity` value here is `0` than it would be changed to `1`.

Reply:
@tgoc99 I am with @whyn07m3, this would be better served with hasOwnProperty or something similar as `= 0 || anything` will always resolve to anything, not sure 0 is a valid value but we should code around it.

Reply:
👍 on Roma's opacity. lets check if its a number here. people do weird things...

Reply:
do we have `isNan(...)` ? seems appropriate here:

```
const opacity = isNan(noteWin.opacity) ? 1 : noteWin.opacity;
```

Reply:
I like this

Reply:
actually I dont, this transitions from 0 to 0 in the default case